tools/clang/test/HLSLFileCheck/hlsl/workgraph/implicit_record_dispatchgrid.hlsl

```diff
@@ -1,30 +1,15 @@
 // RUN: %dxc -T lib_6_8 %s | FileCheck %s
 // RUN: %dxc -T lib_6_8 %s | %D3DReflect %s | FileCheck -check-prefix=REFL %s
 
+// The purpose of this test is to verify the behavior of the compiler on node
+// shaders with no input records
```

Comment:
Yes, and I think we now need to fail this case when no input record is defined in the HLSL entry, yet it is necessary to provide the `SV_DispatchGrid`, due to the use of `[MaxNodeDispatchGrid(...)]`.